webpack.config.js

```diff
@@ -115,7 +115,7 @@ module.exports = {
     module: {
         rules: [
             {
-                test: /\.jsx?$/,
+                test: /\.(?:js|mjs|cjs)x?$/,
```

Comment:
This means we now transpile all JS, not just JSX.
Adding `cjs` and `mjs` aren't necessary just yet, but may help avoid surprises in the future.